jobs/service-fabrik-broker/templates/bin/service-fabrik-broker_ctl.erb

```diff
@@ -27,18 +27,33 @@ case $1 in
         if ! grep -Fxq "\$ModLoad imtcp" /etc/rsyslog.d/10-fabrik_modules.conf ; then echo "\$ModLoad imtcp" >> /etc/rsyslog.d/10-fabrik_modules.conf ; fi
     fi
     /usr/sbin/service rsyslog restart
-    
-    # Start Service-Fabrik-Broker
-    start-stop-daemon --start --quiet -m \
-      --pidfile ${PID_FILE} \
-      --startas ${NODE_CMD} \
-      --chuid vcap:vcap \
-      --background \
-      --no-close \
-      --chdir ${PACKAGE_DIR} \
-      -- ${PACKAGE_DIR}/broker/server.js \
-      1>> ${LOG_DIR}/${OUTPUT_LABEL}.stdout.log \
-      2>> ${LOG_DIR}/${OUTPUT_LABEL}.stderr.log
+
+    if [ ${PERF_TEST_ENABLED} = true ]; then
+        # Start Service-Fabrik-Broker with perf enabled
+        start-stop-daemon --start --quiet -m \
+          --pidfile ${PID_FILE} \
+          --startas ${NODE_CMD} \
+          --chuid vcap:vcap \
+          --background \
+          --no-close \
+          --chdir ${PACKAGE_DIR} \
+          -- --perf-basic-prof \
```

Comment:
I would have done this a bit differently, (which is generally the way we used to do when specifying opts to Java process)
```
OPTS= ""
if [ ${PERF_TEST_ENABLED} = true ]; then
   OPTS="$OPTS -- --perf-basic-prof"
fi
# Similarly keep on enhancing opts and at end do this:
           start-stop-daemon --start --quiet -m \
          --pidfile ${PID_FILE} \
          --startas ${NODE_CMD} \
          --chuid vcap:vcap \
          --background \
          --no-close \
          --chdir ${PACKAGE_DIR} \
          -- ${PACKAGE_DIR}/broker/server.js \
           $OPTS
          1>> ${LOG_DIR}/${OUTPUT_LABEL}.stdout.log \
          2>> ${LOG_DIR}/${OUTPUT_LABEL}.stderr.log
```

Nothing wrong with the below as well. Just the above keeps the code concise and also adding additional options in future is easier. Just a suggestion. 
Still approving the PR.